config/application.rb

```diff
@@ -15,5 +15,9 @@ class Application < Rails::Application
     # Application configuration can go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
+
+    config.action_view.field_error_proc = Proc.new do |html_tag, instance|
+      %Q(#{html_tag}).html_safe
+    end
```

Comment:
[IMO] この部分の処理，何をしているのかぱっと見でわかりにくいから，コメントがあると Better．

e.g. "Rails が自動生成する error フィールドを取り消す"

Reply:
# These means don't Rails enclose error fields by <div class="field-with-errors">
このような感じで大丈夫でしょうか。

Reply:
# To avoid format-brake
# set FALSE to auto-generated error fields by AR generator when `db:migrate`:
# <div class="field-with-errors">
#         <input>...</input> 
# <div>
# => <input>...</input>

とか僕なら書くかな

Reply:
頭よ

Reply:
そのままコピペして、コミットしますね。